cypress-io/cypress
# chore: Add Studio UI to Cypress 10

- Closes #23337

### User facing changelog


This PR adds the Cypress studio UI back into the app with some styling changes. Note that this change will be merged into the `feat/studio-cypress-10` branch to be reviewed all together and merged at a later time.

### Steps to test


1. Add `experimentalStudio: true` your project's Cypress configuration under `e2e`
2. Launch e2e testing for with your project
3. Run a spec
4. Hover over a test in the runner command log and click the magic wand icon
5. Click through the Studio UI and ensure that it is consistent with [the mockup](https://www.figma.com/file/PAOozuCxtfetqP8fUVvg0y/branch/gMEcNAtUnMXhR4tyBLOBVx/Cypress-App?node-id=16373%3A174947)

### How has the user experience changed?


![Screen Shot 2022-08-24 at 4 37 57 PM](https://user-images.githubusercontent.com/14275198/186519053-bb87438a-322c-4c2c-8a80-8c51f0b2dbf5.png)

![Screen Shot 2022-08-24 at 4 38 26 PM](https://user-images.githubusercontent.com/14275198/186519016-9d3307b6-7135-48b7-a744-ca87dca0ee4a.png)

![Screen Shot 2022-08-24 at 4 39 35 PM](https://user-images.githubusercontent.com/14275198/186519128-7083e04f-b5c0-481b-ae9b-4f3e793e3e33.png)

![Screen Shot 2022-08-24 at 4 38 56 PM](https://user-images.githubusercontent.com/14275198/186519143-28c5951d-d302-4afb-aed3-40c7be1e227e.png)

### PR Tasks
Deleting the PR checklist since we'll be handling more testing, documentation, etc when we merge the feature branch.
